elastic/integrations
# [Microsoft Sqlserver] - Ingest instance and server name & query optimisation

- Enhancement


## What does this PR do?

Ingests the instance name and server name.
Fine tuned the query and removed the hardcoding of database names in the response.

Pending
The Kibana visualisation support for filtering the data based on instance name needs to be done

## Checklist



## How to test this PR locally

Run system test and verify the instance and server name fields are getting ingested for both the datastreams.


## Related issues

- Closes #4690 

## Screenshots
